onnxruntime/core/providers/qnn/builder/opbuilder/simple_op_builder.cc

```diff
@@ -109,6 +115,43 @@ Status SimpleOpBuilder::ProcessAlphaAttribute(QnnModelWrapper& qnn_model_wrapper
   return Status::OK();
 }
 
+Status SimpleOpBuilder::ProcessBlockSizeAttribute(QnnModelWrapper& qnn_model_wrapper,
+                                                  const NodeUnit& node_unit,
+                                                  std::vector<std::string>& param_tensor_names) const {
+  NodeAttrHelper node_helper(node_unit);
+  int64_t block_size = node_helper.Get("blocksize", static_cast<int64_t>(0));
```

Comment:
There's now a `node_helper.Get` method for uint32_t. Can use this to avoid casting block_size from int64 to uint32.

```C++
uint32_t block_size = node_helper.Get("blocksize", static_cast<uint32_t>(0));
```